metadata-ingestion/src/datahub/sql_parsing/schema_resolver.py

```diff
@@ -123,6 +123,13 @@ def get_urn_for_table(
         )
         return urn
 
+    def resolve_urn(self, urn: str) -> Tuple[str, Optional[SchemaInfo]]:
+        schema_info = self._resolve_schema_info(urn)
+        if schema_info:
+            return urn, schema_info
```

Comment:
add a test for this in `test_schemaresolver.py`

Reply:
done